P/pandoc_crossref/build_tarballs.jl

```diff
@@ -0,0 +1,50 @@
+using BinaryBuilder
+
+# Collection of pre-build pandoc binaries
+name = "pandoc-crossref"
+
+# Actually, this should be v"0.3.6.4".
+version = v"0.3.6"
+crossref_ver = "0.3.6.4"
+pandoc_ver = "2.9.2.1"
+
+url_prefix = "https://github.com/lierdakil/pandoc-crossref/releases/download/v$crossref_ver/pandoc-crossref"
+sources = [
+    ArchiveSource("$url_prefix-Linux-$pandoc_ver.tar.xz", "57e8f71d46c401daf0a3247b7405b49503b836e93893730e6e5907f6cb2c0885"; unpack_target = "x86_64-linux-gnu"),
+    ArchiveSource("$url_prefix-macOS-$pandoc_ver.tar.xz", "b63ac830d7164279eb80041cd5f793e6b611ce1a701a86308ce5b2a3a99d33b2"; unpack_target = "x86_64-apple-darwin14"),
+    # Unknown archive format.
+    # ArchiveSource("$url_prefix-Windows-$pandoc_ver.7z", "3e943fcdd3f91951fe18f9900e136ccb9ea810b2582f4bd929760357eaf83ef4"; unpack_target = "x86_64-w64-mingw32"),
```

Comment:
This can be downloaded as
```suggestion
   FileSource("$url_prefix-Windows-$pandoc_ver.7z", "3e943fcdd3f91951fe18f9900e136ccb9ea810b2582f4bd929760357eaf83ef4"; unpack_target = "x86_64-w64-mingw32"),
```
but you'll need to unpack it manually in the recipe with `7z`

Reply:
Check. I'll do that.